src/rpc/handlers/LedgerEntry.hpp

```diff
@@ -310,10 +311,11 @@ class LedgerEntryHandler {
      *
      * @param input The input data for the command
      * @param ctx The context of the request
+     * @param include_deleted Whether to include deleted entries
      * @return The result of the operation
      */
     Result
-    process(Input input, Context const& ctx) const;
```

Comment:
same with this, you need to modify the Output class that declares the response params